exercises/practice/hamming/.meta/example.8th

```diff
@@ -3,18 +3,26 @@
 ;
 
 : distance \ s1 s2 | n
-	empty-string? if drop drop null ;then
-	swap
-	empty-string? if drop drop null ;then
-	
+	\ if len s1 is zero 
+	\ 	if len s2 is zero 
+	\ 		return zero
+	\	return null
+	empty-string? 
+	if 
+	 	swap empty-string? 
+		if \ both empty
+			drop drop 0 ;; 
+		then
+		drop drop null ;; 
+	then
+
```

Comment:
Actually, you don't need any of this. Comparing 2 empty strings is accomplished on lines 25 to 27.

Also, `tests.8th` needs to be changed -- [the first test](https://github.com/exercism/8th/blob/fd07fddf474b855ed04c4c12620bb2c4247e4784/exercises/practice/hamming/test.8th#L8) should be `0 equal?`

Reply:
Okay, fixed that. Thanks @glennj